### PR TITLE
Facelift: Card reviews

### PR DESCRIPTION
**Pull request by lenasmayor**
Closes XTD-184 and XTD-186

- Synced sandbox card and list view: Change icon and add 'synced' tag
image

- Repository card and list view: Change icon and adjust paddings
image

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6942/commits/6689636a81f43e892fb8b0597e48175fc0bf1575 is at https://pr6942.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Components/Repo/RepoListItem.tsx [hunk 1]
```diff
@@ -63,9 +76,21 @@ export const RepoListItem = ({
...
+          <Text
+            size={3}
+            paddingX={4}
+            paddingY={2}
+            css={{
+              color: '#C2C2C2',
+              backgroundColor: '#1D1D1D',
+              borderRadius: 99,
+              ':hover, :focus, :focus-within': {
+                backgroundColor: '#303030',
```
Comment (alexnm) [inline packages/app/src/app/pages/Dashboard/Components/Repo/RepoListItem.tsx:89]: I would only change the backgroundColor on `hover` and add a `focus` outline with purple, like in the sandbox cards.

**Review by alexnm — APPROVED** [review]
Left a single comment on the focus/hover difference. but feel free to merge this and we can add that separately

File: packages/app/src/app/pages/Dashboard/Components/Repo/RepoListItem.tsx (hunk 1 above)
Comment (lenasmayor) [reply packages/app/src/app/pages/Dashboard/Components/Repo/RepoListItem.tsx:89]: This only changes the color of the Synced tab on hover so it doesn't fade in the background. The background color changes for the entire component on `hover` and `focus` had already been implemented  on line 46-48.

**Pull request merged**